CPAC/pipeline/cpac_pipeline.py

```diff
@@ -363,17 +360,21 @@ def run_workflow(sub_dict, c, run, pipeline_timing_info=None, p_name=None,
         System time of start:      {run_start}
         System time of completion: {run_finish}
 
-"""
+"""  # noqa E501
 
-    logger.info(information.format(
+    logger.info('%s', information.format(
         run_command=' '.join(['run', *sys.argv[1:]]),
         cpac_version=CPAC.__version__,
         cores=c.pipeline_setup['system_config']['max_cores_per_participant'],
         participants=c.pipeline_setup['system_config'][
             'num_participants_at_once'],
         omp_threads=c.pipeline_setup['system_config']['num_OMP_threads'],
         ants_threads=c.pipeline_setup['system_config']['num_ants_threads'],
-        max_cores=max_core_usage
+        max_cores=max_core_usage,
+        random_state=(
```

Comment:
Similar to the above, we probably want to be consistent with language. To my understanding, this is more accurately the random seed than the random state. Do you agree?

Reply:
Yep, this is where the placeholder variable is defined. I'll change the placeholder variable name to `random_seed`